apps/backend/src/mcp/tools/system_monitoring.py

```
@@ -983,6 +962,286 @@ async def get_system_logs(
         )
 
 
+async def get_drive_stats(
+    device: str,
+    drive: str | None = None,
+    timeout: int = 60
+) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)